website/blog/2020-08-04-ariadne-0-12-0.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Ariadne 0.12.0
+---
+
+This one took us a while, but Ariadne 0.12 is now available!
+
+Ariadne 0.12 is a feature release that adds support for custom query validators, adds query cost validator that enables developers to limit maximum complexity of GraphQL query that can be executed by the server and adds option for disabling GraphQL introspection and GraphQL playground on servers.
+
+This release also locks required GraphQL core version at 3.0. Support for 3.1 is coming in future version.
+
+Finally, this release also adds `extensions` option to Django view and fixes reported bug in `ScalarType` default `parse_literal` producing error when GraphQL query contained both literal and query variables.
+
+<!--truncate-->
+
+
+## Custom query validators and query cost validator
+
+It is now possible to implement custom GraphQL query validators. This is an advanced feature intended for developers who need to block malicious GraphQL queries from being executed by their server.
+
+Out of the box Ariadne includes [query cost validator](/docs/query-validators) that enables developers to assign costs values to GraphQL fields in the schema and limit size or complexity of single GraphQL query that may be executed by the server.
+
+
+## Added `introspection` option in ASGI, WSGI apps and Django view
+
+One of most requested features for Ariadne was option to disable GraphQL Playground. This request had its basis on security concerns, however disabling GraphQL Playground in Ariadne would not prevent users from using local GraphQL API client like Insomnia or desktop version of GraphQL Playground to continue interacting with the API.
+
+This is because when users request ability to disable GraphQL Playground, they in fact seek option to disable introspection: GraphQL feature enabling clients to query GraphQL server to learn what types, fields and mutations are implemented by it.
```

Comment:
```suggestion
This is because when users request the ability to disable GraphQL Playground, they in fact seek an option to disable introspection, a GraphQL feature enabling clients to query a GraphQL server to learn what types, fields, and mutations are implemented by its schema.
```